packages/system/kubeovn/charts/kube-ovn/templates/ovn-CR.yaml

```
@@ -248,7 +290,34 @@ rules:
+      - authentication.k8s.io
+    resources:
+      - tokenreviews
+    verbs:
+      - create
+  - apiGroups:
+      - authorization.k8s.io
+    resources:
+      - subjectaccessreviews
+    verbs:
+      - create
+  - apiGroups: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)